doc/windows/package-manager/winget/upgrade.md

```diff
@@ -22,23 +22,41 @@ The **upgrade** command requires that you specify the exact string to upgrade. I
+| **--include-unknown**            | Attempt to upgrade a package even if the package's current version is unknown.                                              |
+| **--purge**                       | Deletes all files and directories in the package directory (portable).                                                      |
+| **--custom**                      | Arguments to be passed on to the installer in addition to the defaults.                                                     |
+| **--scope**                       | Select installed package scope filter (user or machine).                                                                    |
+| **-a, --architecture**            | Select the architecture to install.                                                                                         |
+| **--locale**                      | Locale to use (BCP47 format).                                                                                               |
+| **--ignore-security-hash**        | Ignore the installer hash check failure.                                                                                    |
+| **--ignore-local-archive-malware-scan** | Ignore the malware scan performed as part of installing an archive-type package from a local manifest.                  |
+| **--accept-package-agreements**   | Accept all license agreements for packages.                                                                                 |
+| **--accept-source-agreements**    | Accept all source agreements during source operations.                                                                      |
+| **--header**                      | Optional Windows-Package-Manager REST source HTTP header.                                                                   |
+| **-r, --recurse, --all**         | Upgrade all installed packages to the latest version if available.                                                           |
+| **--pinned,--include-pinned**    | Upgrade packages even if they have a non-blocking pin.                                                                      |
+| **--uninstall-previous**         | Uninstall the previous version of the package during the upgrade.                                                            |
+| **--wait**                        | Prompts the user to press any key before exiting.                                                                           |
+| **--logs,--open-logs**           | Open the default logs location.                                                                                             |
+| **--verbose,--verbose-logs**     | Enable verbose logging for winget.                                                                                          |
+| **--disable-interactivity**      | Disable interactive prompts.                                                                                                |
+
```

Comment:
There are two more arguments that are missing from this Can you add those as well?

` --installer-type                     Select the installer type`
  
`--skip-dependencies                  Skips processing package dependencies and Windows features`